MAUI/Picker/dealing-with-header-and-footer.md

```diff
@@ -0,0 +1,314 @@
+        </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    using Syncfusion.Maui.Picker;
+    using System.Collections.ObjectModel;
+    namespace Picker_29;
+    
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            SfPicker picker = new SfPicker();
+            picker.HeaderView.TextStyle = new PickerTextStyle()
+            {
+                TextColor = Colors.Gray,
+                FontSize = 18,
+                FontAttributes = FontAttributes.Italic
+            };
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Enable or disable footer
+
+SfPicker allows enabling or disabling the footer section by setting  `SfPicker.FooterView.Height` property grater than 0. Default value of `SfPicker.FooterView.Height` property is 0.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+             x:Class="Picker_29.MainPage">
+
+    <sfPicker:SfPicker x:Name="picker">
+        <sfPicker:SfPicker.FooterView>
+            <sfPicker:PickerFooterView Height="50"/>
+        </sfPicker:SfPicker.FooterView>
+    </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    picker.FooterView.Height= 50;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Footer view customization
+
+SfPicker allows customizing background, text and text style style of the `ok` and `cancel` button and also visibility of the the `ok` button.
+
+### Background
+
+Header’s background color can be customized by setting `SfPicker.HeaderView.Background` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+            xmlns:sfPicker="clr-namespace:Syncfusion.Maui.Picker;assembly=Syncfusion.Maui.Picker"
+            x:Class="Picker_29.MainPage">
+
+    <sfPicker:SfPicker x:Name="picker">
+        <sfPicker:SfPicker.FooterView>
+            <sfPicker:PickerFooterView Background="Gray"/>
+        </sfPicker:SfPicker.FooterView>
+    </sfPicker:SfPicker>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+    picker.FooterView.Background=Brush.Gray;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Buttons customization
+
+SfPicker allows enabling or disabling the `ok` button by setting SfPicker.FooterView.ShowOkButton property to True or False. Default value of SfPicker.FooterView.ShowOkButton property is True. We can customize the text of the `ok` and and `cancel` buttons.
```

Comment:
SfPicker allows enabling or disabling the `ok` button by setting SfPicker.FooterView.ShowOkButton property to True or False. Default value of SfPicker.FooterView.ShowOkButton property is True. We can customize the text of the `ok` and `cancel` buttons.

Reply:
Content modified